### ISSUE TITLE
Higher order fit for LROC NAC

### ISSUE THREAD

**Issue by meganhenriksen** [USGS-Astrogeology/usgscsm#355]
Stereo images taken post 2018 are taken without the MIMU and have movement in the CK that doesn't seem to be quite accounted for in the CSM (though it is much better than the generic pushbroom model (RMS 0.55 vs 1.078). Jesse mentioned that you might be able to do a higher order fit for (something??) that might help us. If we could try that, I would be very grateful. 

We've been currently testing this with this NAC pair: M1371324649, M1371331677

Here are the plots of the CK motion. We're also working on some residual plots. 
https://drive.google.com/drive/folders/1cK0IpCfJNdXIyweC_0bs0htzvZiCIG6a?usp=sharing

**Here's a brief history of our attempts to work with this pair.** 
Added a bunch of tie points (including ¾-ways to seams) to the project.
Solved 0.78, saved. Ran NGATE DTMs at 3.0m GSD
**Weird terrain correlated with high tie point residuals at 17.8N. (~ halfway between top/bottom)**
Left side had some usable terrain, right was unusable.
Edited out weird terrain, registered remaining terrain of left side.
Removed high residual points.
Tried making more DTMs to register with LOLA, none of which has been usable
Most recent solve: (RMS: 0.55, X: 5.26, Y: 9.09, Z: 0.58), saved.

I realize this is somewhat vague, so please let me know what questions I can try to answer.

**Comment by jessemapel** [comment]
Sorry it's taken a while to get down to this issue. I'm fine with just keeping this discussion going as "how do we better adjust these images?"

I would try adding the angle bias for the other angles at around the same accuracy and see what that gets you.

**Comment by Kelvinrr** [comment]
@meganhenriksen I am struggling to replicate this in a way similar to the examples you provided. 

I think I tracked down the same image you are using in the archives and opened up the CKs to plot the orientations. I am seeing a little bit of jitter in the orientations but not to the same magnitude you are seeing in your visualizations. 

You can see what I did to get try to match your plots here. There is a lot of jitter in one dimension of rotation, so maybe it's simply a different representation of the same issue:

https://gist.github.com/Kelvinrr/614a876321b470fd87a2fb735fc7b529 

Im using a library that is syntax sugar on top of cspice calls to search for the kernels, so I believe these are the values stored directly on the CK's segments.

**Comment by AustinSanders** [comment]
I updated Kelvin's visualization notebook using the new reference frame (LRO_REFERENCE), which seems to have reproduced the results described in the provided plots.

The updated notebook can be found at https://gist.github.com/AustinSanders/3138a954178d9bd63a5afad202c68f1c

<img width="864" alt="Screenshot 2023-12-28 at 12 12 12 PM" src="https://github.com/DOI-USGS/usgscsm/assets/3751180/8e704b4e-a37d-4e9b-98b2-97443ca204a1">

**Comment by oleg-alexandrov** [comment]
This is likely jitter, and likely the current linescan sensor for LRO NAC has enough expressivity. One can go as far as assigning a single position and pose per line if one wishes so (when it will blow up of course).

The problem is not being able to do a higher order fit but to find the coefficients for it (regardless of how the model expresses it).

The natural in-house attempt is jigsaw with a higher cksolve degree, but that is the same higher-order fit already shown insufficient here. A different class of tool is ASP jitter_solve, which can minimize the jitter, though much depends on whether the sensors provide orthogonal-enough data to disambiguate it.

In short, likely nothing will happen here for the foreseeable future, as USGSCSM is likely not the package for this. No amount of modeling here will help without some joint sensor optimization, which belongs in the bundle adjustment layer rather than in the sensor model.
